src/cluster.c

```diff
@@ -5637,6 +5637,49 @@ sds genClusterInfoString() {
     return info;
 }
 
+/* Add detailed information of a node to the output buffer of the given client for keyshard command */
+void getKeyShardReply(client *c, clusterNode *node) {
```

Comment:
Not clear why we would just call `addShardReplyForClusterShards` for the single shard.

Reply:
7) "endpoint"
          8) "127.0.0.1"
         11) "replication-offset"
         12) (integer) 72156
         13) "health"
         14) "online"